toolchain/check/handle_let_and_var.cpp

```diff
@@ -386,6 +386,12 @@ auto HandleParseNode(Context& context, Parse::VariableDeclId node_id) -> bool {
     }
     return true;
   }
+  if (context.GetCurrentScopeAs<SemIR::InterfaceDecl>()) {
+    CARBON_DIAGNOSTIC(VarInInterfaceDecl, Error,
+                      "interfaces cannot have `var` members");
```

Comment:
The style guide wants to say what occurred, rather than what cannot occur: https://github.com/carbon-language/carbon-lang/blob/40f3de2c0726983c7c41200eddd479ceec00be5e/toolchain/docs/diagnostics.md#diagnostic-message-style-guide

Maybe "`var` binding found in `interface`"?

Reply:
OK, done.